app/models/dialog_field_importer.rb

```diff
@@ -12,6 +12,13 @@ def import_field(dialog_field_attributes)
       resource_action_attributes = dialog_field_attributes.delete("resource_action")
       resource_action = ResourceAction.new(resource_action_attributes)
       dialog_field = dialog_field_type_class.new(dialog_field_attributes.merge("resource_action" => resource_action))
+      if dialog_field_attributes["type"] == "DialogFieldTagControl"
+        dialog_field_attributes["options"].delete(:category_id)
```

Comment:
@eclarizio This will break older dialog exports since they will not contain the `category_name` and therefore will always end up with an empty category on the tag control field even when imported into the same appliance.  (Use case would be backup/restore)

The category lookup should still use the `category_id ` if the `category_name` field does not next in the import data.  If the `category_name` key exists I think we should always use that for the lookup.